### DISCUSSION TITLE
RFC: workflow hints section

### DISCUSSION THREAD

**Discussion by jdidion** [openwdl/wdl#417] [Requests for Comment]
We are beginning to have features in the development spec that leverage workflow-level hints to the runtime engine, e.g. the `allowNestedInputs` flag described in https://github.com/openwdl/wdl/pull/359. Such hints do not really belong in `meta`. I propose that we add a `hints` section to workflow that has identical semantics to task hints.

**Comment by patmagee** [comment]
As I Mentioned in the linked PR, I think this is ultimately a good Idea. we already have 1 use case for `hints` being in the `workflow` as opposed to just the `task` which likely means there are going to be other uses that we come across

**Comment by rhpvorderman** [comment]
This would indeed be best. It was only added in meta because of a lack of a hints section.
